### ISSUE TITLE
TwilioPricingClient.getVoiceCountry not working on countries with toll free number type

### ISSUE THREAD

**Issue by ivanmb** [twilio/twilio-java#247]
The VoiceCountry class tries to pull the NumberType enum value with a valueOf, but the internal code contains a whitespace. I inspired the solution on the PhoneNumberCountry class, that replaces the whitespace with an underscore. 

**Version: 6.3.0**

### Code Snippet
```java
VoiceCountry voiceCountry = this.twilioPricingClient.getVoiceCountry(isoCountryCode);
voiceCountry.getInboundCallPrices() 
```

### Exception / Log
```
java.lang.IllegalArgumentException: No enum constant com.twilio.sdk.resource.instance.pricing.NumberType.TOLL FREE
```

I forked and fixed it adding a new test case, that you can find here: https://github.com/ivanmb/twilio-java/commit/ca409773374e667711910e73e692ecf7c181d9b7.

Please confirm, and I create the pull request as soon as posible. I need this for a production environment.

Thanks!

**Comment by carlosdp** [comment]
Nice catch, yea could you open a PR please? Thanks!

**Comment by carlosdp** [comment]
In the meantime, if you need this in order to deploy, you should probably deploy your fork to Nuget and use that until the PR is merged.

**Comment by ivanmb** [comment]
Thanks Carlos for the quick response! How long do you think it could take until this is released as perhaps a minor version like 6.3.1?

**Comment by Ragil** [comment]
@ivanmb We released a fix to version 5.10.0

Version 6.3.0 was an accidental release with a typo in the version number. Also if you are interested, we have a release candidate for a completely new helper library that is far more ergonomic, performant, and consistent. It is released under version 7.x.
